Cargo.toml

```diff
@@ -50,7 +51,7 @@ backend_drm = ["drm", "failure"]
 backend_drm_legacy = ["backend_drm"]
 backend_drm_gbm = ["backend_drm", "gbm", "image"]
 backend_drm_egl = ["backend_drm", "backend_egl"]
-backend_egl = ["gl_generator"]
+backend_egl = ["gl_generator", "wayland-egl"]
```

Comment:
I think `waykand-egl` should instead be introduced as a dependency to `backend_winit`, since this is the only part of Smithay that needs it (and it already depends on `backend_egl`).

Reply:
Well, why not? Done.